simulators/common/testmanager.go

```diff
@@ -143,6 +143,7 @@ func (manager *TestManager) CreateNetwork(testSuite TestSuiteID, networkName str
 	// create network
 	network, err := manager.dockerClient.CreateNetwork(docker.CreateNetworkOptions{
 		Name:           networkName,
+		Driver:         "bridge",
```

Comment:
@fjl should this be a default, or should this be specified by the user? Maybe at one point we'll want two totally isolated networks or something? Wdyt?

Reply:
I think it's OK to just put this in for now, we can always add more config possibilities later when we actually need it.